Exercise_A/index.html

```diff
@@ -16,7 +16,9 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <a href="./about.html">
+      <img src="https://picsum.photos/id/1062/500/500" alt="A cute little doggy">
+    </a>  
```

Comment:
Nice work you have followed the correct coding conventions.
I like how you have added a go back button. Great job